nautobot/docs/plugins/development.md

```diff
@@ -903,9 +903,12 @@ class Animal(BaseModel):
 
 #### Creating Your Own GraphQL Type Object
 
++/- 1.6.1
+    A new base class was introduced for Nautobot GraphQL object types: `nautobot.core.graphql.types.OptimizedNautobotObjectType`. This class inherits from `graphene_django_optimizer.OptimizedDjangoObjectType` and adds generic Nautobot specific functionality.
```

Comment:
Should this also have some verbiage steering developers to use this as their base class? 

```suggestion
    A new base class was introduced for Nautobot GraphQL object types: `nautobot.core.graphql.types.OptimizedNautobotObjectType`. This class inherits from `graphene_django_optimizer.OptimizedDjangoObjectType` and adds generic Nautobot specific functionality. All Nautobot GraphQL object types should inherit from this base class.
```